kythe/extractors/BUILD

```diff
@@ -198,6 +198,7 @@ extractor_action(
     data = ["//external:vnames_config"],
     env = {
         "LD_LIBRARY_PATH": "external/rust_linux_x86_64/lib:external/rust_darwin_x86_64/lib",
+        "KYTHE_CORPUS": "kythe.io",
```

Comment:
Why is this needed?

Reply:
The extractor requires the "KYTHE_CORPUS" environment variable to be set. I don't think this was caught previously because the extractor hasn't been run on the Kythe codebase in a while. Another option is to simply set a default corpus in the extractor.